app/views/create_new_user_requests/_request_details.html.erb

```diff
@@ -38,6 +38,24 @@
       </span>
     </div>
 
+    <div class="form-group">
+      <span class="form-label">
+        <%= f.label :new_or_existing_user, class: "control-label" do %>
```

Comment:
Probably out of scope for this PR, but I wonder if the questions could also be added into the configuration file, so everything is configured there?

Reply:
Feels sensible - there or the en.yml file. But yeah, maybe one for the backlog to do that here and in other forms